SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -84,6 +85,11 @@ void USpatialGDKSettings::PostEditChangeProperty(FPropertyChangedEvent& Property
 		ValidateOffloadingSettings();
 	}
 
+	if (PropertyName == GET_MEMBER_NAME_CHECKED(USpatialGDKSettings, bEnableOffloading))
```

Comment:
Reminder to remove this if we decide to remove the toggle in ULevelEditorPlaySettings.